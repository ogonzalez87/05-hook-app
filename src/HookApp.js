import React from 'react'

export const HookApp = () => {
    return (
        <div>
            <h1>Hola Mundo!</h1>

            <h2>Hola de nuevo temp</h2>
        </div> 
    )
}
