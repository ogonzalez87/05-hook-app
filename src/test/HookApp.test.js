// import React from 'react';
import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe("HookApp", () => {
  test("should HookApp works", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
