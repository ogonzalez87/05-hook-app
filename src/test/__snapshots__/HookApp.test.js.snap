// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp should HookApp works 1`] = `
<div>
  <h1>
    Hola Mundo!
  </h1>
</div>
`;
